instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -182,6 +183,11 @@ def _traced_execute_command(func, instance, args, kwargs):
         query = _format_command_args(args)
         name = _build_span_name(instance, args)
 
+        if context.get_value(
+            _SUPPRESS_INSTRUMENTATION_KEY
+        ):
```

Comment:
There is a function called `is_instrumentation_enabled` in the `opentelemetry.instrumentation.utils` module that should be used instead of accessing _SUPRESS_INSTRUMENTATION_KEY directly.

Reply:
Without knowing what arguments you are passing and what error you are getting it's hard to give any specific suggestion.

Reply:
No arguments, https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2330